internal/docs/readme.go

```diff
@@ -17,27 +17,56 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
+// ReadmeFile contains file name and status of each readme file.
+type ReadmeFile struct {
+	FileName string
+	UpToDate bool
+	Error    error
+}
+
+// AreReadmesUpToDate function checks if all the .md readme file are up-to-date.
+func AreReadmesUpToDate() ([]ReadmeFile, error) {
+	packageRoot, err := packages.MustFindPackageRoot()
+	if err != nil {
+		return nil, errors.Wrap(err, "package root not found")
+	}
+
+	files, err := ioutil.ReadDir(filepath.Join(packageRoot, "_dev", "build", "docs"))
+	if err != nil && !os.IsNotExist(err) {
+		return nil, errors.Wrap(err, "reading directory entries failed")
+	}
+
+	var readmeFiles []ReadmeFile
+	for _, f := range files {
+		fileName := f.Name()
+		ok, err := isReadmeUpToDate(fileName, packageRoot)
+		readmeFile := ReadmeFile{
+			FileName: fileName,
+			UpToDate: ok,
+			Error:    err,
+		}
+		readmeFiles = append(readmeFiles, readmeFile)
+	}
+	return readmeFiles, nil
```

Comment:
I think you should return an error if processing of any readme file failed.

Reply:
done!